v0/destinations/splitio/data/EventConfig.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "destKey": "eventTypeId",
+    "sourceKeys": [
+      "eventTypeId",
+      "properties.eventTypeId",
+      "event",
+      "name",
+      "traits.eventTypeId",
+      "context.traits.eventTypeId"
+    ],
+    "metadata": {
+      "type": "toString"
+    },
+    "required": true
+  },
+  {
+    "destKey": "environmentName",
+    "sourceKeys": [
+      "environmentName",
+      "properties.environmentName",
+      "traits.environmentName",
+      "context.traits.environmentName"
+    ],
+    "metadata": {
+      "type": "toString"
+    },
+    "required": false
+  },
+  {
+    "destKey": "trafficTypeName",
+    "sourceKeys": [
```

Comment:
again same as environment name why are you mapping here?

Reply:
eliminated trafficType, will take it from config